Resources/Prototypes/DeltaV/Catalog/Fills/Crates/food.yml

```diff
@@ -29,3 +29,119 @@
           amount: 1
         - id: FoodBoxDonkpocketDink
           prob: 0.1
+
+- type: entity
+  id: CrateHydroCoDairy
+  parent: CratePlastic
+  name: HydroCo Dairy crate
+  description: A shipment of powdered dairy products from HydroCo. Contains powdered milk and soy milk.
+  components:
+    - type: StorageFill
+      contents:
+        - id: PaperCrateHydroCoConsumption
+          amount: 1
+        - id: SpoonPlastic
+          amount: 2
+        - id: ReagentTinPowderedMilk
+          amount: 2
+        - id: ReagentTinPowderedMilkSoy
+          amount: 2
+
+- type: entity
+  id: CrateHydroCoJuice
+  parent: CratePlastic
+  name: HydroCo Juice crate
+  description: A variety pack of powdered juice products from HydroCo.
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:AllSelector
+        children:
+        - !type:NestedSelector
+          tableId: FillHydroCoJuiceCitrus
+        - !type:NestedSelector
+          tableId: FillHydroCoJuiceBerry
+        - !type:NestedSelector
+          tableId: FillHydroCoJuiceTropical
+        - !type:NestedSelector
+          tableId: FillHydroCoJuiceGarden
+        - id: SpoonPlastic
+          amount: !type:ConstantNumberSelector
+            value: 2
+        - id: PaperCrateHydroCoConsumption
+          amount: !type:ConstantNumberSelector
+            value: 1
+
+- type: entityTable
```

Comment:
Modular tables like this works but they *could* be condensed. See: [comment](https://discord.com/channels/968983104247185448/1307145855199084644/1308253511133171772) in Discord.